vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -179,6 +197,9 @@ func (s *GenericAPIServer) UnprotectedHandler() http.Handler {
 	// when we delegate, we need the server we're delegating to choose whether or not to use gorestful
 	return s.Handler.Director
 }
+func (s *GenericAPIServer) SwaggerAPIContainers() []*restful.Container {
+	return s.swaggerAPIContainers
```

Comment:
do the  `append` with the delegation target's containers here. Would be pretty obvious then and we can get rid of the terrible 5 liner comments above.